ts/sagas/wallet.ts

```diff
@@ -345,6 +349,42 @@ function* addCreditCard(
   yield put(navigateTo(ROUTES.WALLET_HOME));
 }
 
+function* deleteWallet(
+  walletId: number,
+  pagoPaClient: PagoPaClient
+): Iterator<Effect> {
+  try {
+    yield put(walletManagementSetLoadingState());
+    const response:
+      | BasicResponseTypeWith401<undefined>
+      | undefined = yield call(
+      fetchWithTokenRefresh,
+      (token: string) => pagoPaClient.deleteWallet(token, walletId),
+      pagoPaClient
+    );
+    if (response !== undefined && response.status === 200) {
+      // wallet was successfully deleted
+      console.warn("Card successfully deleted!"); // tslint:disable-line no-console
```

Comment:
IIRC the behavior for these non-invasive messages is to show a toast. Looking around, I have found [this](https://github.com/Wolox/react-native-redux-toast) component, which is already integrated with redux and makes the displaying of toasts a breeze. I will be creating a separate PR to introduce it in those places that need it (e.g. scan an incorrect QR code, add/delete wallet)

Reply:
NativeBase already has a [toast component](https://docs.nativebase.io/Components.html#toast-def-headref)

Reply:
I would use Native Base Toast component, and avoid adding a new library at this time, we need @matteodesanti to think about the UI/UX of this message - so now minimize the code